contracts/eth/nearprover/UPGRADE.md

```diff
@@ -0,0 +1,35 @@
+# Upgradability
+
+This file describes how to upgrade the bridge address in NearProver contract.
+
+## Configuration
+
+1. Create `.env` file inside `eth-custodian` directory: `$ touch .env`.
+
+2. Add to the file your Infura API key:
+`$ echo "INFURA_API_KEY=YOUR_INFURA_API_KEY_HERE" >> .env` <br/>
+
+3. Add to the file Ethereum Private key:
+`$ echo "ETH_PRIVATE_KEY=YOUR_ETH_PRIVATE_KEY_HERE" >> .env`
+
+## Mock deployment
```

Comment:
I would like to introduce some extra steps here as follow: 
- Integrating [slither](https://github.com/crytic/slither) as a smart contract static analysis tool.
- Apply the [upgradeability-check](https://github.com/crytic/slither/wiki/Upgradeability-Checks) detector.
- Finally logging the analysis results in a report attached to that upgrade.

This is just preventive mechanism against future upgradeability failure.

Reply:
@0x3bfc we are not talking about the upgradability as a proxy here. Here we are talking about changing the reference to the light client inside the existing smart contract using `AdminControlled` interface. So I agree we need `slither` but `upgradability-check` detector is not relevant for this specific PR.

Reply:
It make sense .... Thanks @sept-en 👍